lib/funky/page.rb

```diff
@@ -0,0 +1,41 @@
+module Funky
+  class Page < FunkyObject
+
+    # @return [String] the username.
```

Comment:
@bruce-fs This documentation is not super-helpful. Instead you can copy/paste here what the Facebook docs say about this field. So this could be

```ruby
# @return [String] The alias of the Page. For example, for 
#    www.facebook.com/platform the username is 'platform'.
# @see https://developers.facebook.com/docs/graph-api/reference/page/
def username
```

Same with the other methods